modules/apps/27-interchain-accounts/controller/client/cli/query.go

```diff
@@ -11,6 +11,40 @@ import (
 	"github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/controller/types"
 )
 
+// GetCmdQueryInterchainAccount returns the command handler for the controller submodule parameter querying.
+func GetCmdQueryInterchainAccount() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "interchain-account [owner] [connection-id]",
+		Short:   "Query the interchain account address for a given owner on a particular connection",
+		Long:    "Query the controller submodule for the interchain account address for a given owner on a particular connection",
+		Args:    cobra.ExactArgs(2),
+		Example: fmt.Sprintf("%s query interchain-accounts controller interchain-account connection-0 cosmos1layxcsmyye0dc0har9sdfzwckaz8sjwlfsj8zs", version.AppName),
```

Comment:
fix ordering in example string: 

```suggestion
		Example: fmt.Sprintf("%s query interchain-accounts controller interchain-account cosmos1layxcsmyye0dc0har9sdfzwckaz8sjwlfsj8zs connection-0", version.AppName),
```